src/context/ToolboxContext/index.tsx

```diff
@@ -1,11 +1,10 @@
 import React, { createContext, useEffect, useReducer } from 'react';
 
-// eslint-disable-next-line import/no-unresolved
 import { FeatureCollection } from 'geojson';
 
 import { useFilters } from '@/hooks/useFilters';
 import { getStacItems } from '@/services/stac';
-import { Collection, Feature as StacFeature } from '@/typings/stac';
+import { Collection, StacItem } from '@/typings/stac';
```

Comment:
The point of putting typings in the top-level `typings` directory is to make them globally usable and not have to import them, or your as well keeping them closer to where used, at least that is the general consensus.

This is what the map and vitest-globals type files under the `typings` directory do and it would be good to keep consistent, if we want a typings file that isn't global, then I suggest we add something to `utils` directory.

The only time I ever import types are those generated by 3rd party libs, or created by zod when formatting Data types. I find this very clean, no matter, whichever style is chosen, we should be consistent.

Interested to hear your thoughts.

Reply:
I wasn't sure where I was going wrong as I updated the `stac.ts` to `stac.d.ts` and then the code wasn't able to find these types anymore.

I'm new to Typescript, so there's fundamentals I'm missing.

Reply:
ah right, no worries. first off all files should be named `*.d.ts` as a convention, so I suggest renaming the `stac.ts` to `stac.d.ts` The `d` stands for **definitions** so it is clear to users that only types/interfaces will be there.

If you use `map.d.ts` as your guide, I don't think you can go far wrong, it is really simple and only has 2 types, focus on the `MapConfig type as I think I removed references to the `Layer` type but forgot to remove it.

first export an empty object, I forget the exact reason now but it is necessary.

Wrap any types/interfaces in `declare global {...}` and that should be it.

If you do run into trouble, it could be that you need to add your `*.d.ts` file to `tsconfig.json` so that VS Code can find it, but I've not had to do that for the `map.d.ts` types.

Reply:
Merged in seconds before you posted this. I will come back to this branch/PR and make these changes however as I would like to learn.